src/main/java/org/broadinstitute/consent/http/db/DACUserDAO.java

```diff
@@ -75,7 +75,7 @@ void updateDACUser(@Bind("email") String email,
     Integer verifyAdminUsers();
 
     @Mapper(DACUserRoleMapper.class)
-    @SqlQuery("select u.*, r.roleId, r.name, du.status from dacuser u inner join user_role du on du.user_id = u.dacUserId inner join roles r on r.roleId = du.role_id where r.name = :roleName and du.email_preference = :emailPreference")
+    @SqlQuery("select u.*, r.roleId, r.name, du.status from dacuser u inner join user_role du on du.user_id = u.dacUserId inner join roles r on r.roleId = du.role_id where r.name = :roleName and u.email_preference = :emailPreference")
```

Comment:
I think this would be more readable if `dacuser` was aliased with `du`, `user_role` with `ur`, and keeping `roles` as `r`.

Reply:
Will do.